mailpoet/assets/css/src/components-automation-analytics/header.scss

```diff
@@ -7,6 +7,10 @@
   .woocommerce-filters-filter {
     width: 100%;
 
+    .components-dropdown__content .components-popover__content {
+      padding: 0
+    }
+
```

Comment:
Not really related to this PR. This aligns the date picker popover more to the appearance in WooCommerce itself.